dev/io.openliberty.microprofile.metrics.internal.5.0.monitor/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Per https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines, should the project be named as `io.openliberty.microprofile.metrics.5.0.internal.monitor`?